src/github.com/matrix-org/dendrite/clientapi/auth/auth.go

```diff
@@ -27,6 +29,13 @@ import (
 	"github.com/matrix-org/util"
 )
 
+// UnknownDeviceID is the default device id if one is not specified.
+var UnknownDeviceID = "unknown-device"
```

Comment:
> Repeated logins clobber access tokens for now (so there is only ever 1 valid access_token at any one time, meaning multi-device doesn't work). This is keyed off the device ID, which we don't yet parse out the request.

I'm aware this deviates from Synapse for now. I would rather we didn't amass a huge list of *valid* access tokens for an account, so I'm limiting it to 1 unknown device for now. Clients should be giving us device IDs.